packages/mobile/src/components/LocaleDebug/LocaleDebug.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { Text, ScrollView, StyleSheet } from 'react-native'
+import { DateTime } from 'luxon'
+
+export const LocaleDebug = () => {
```

Comment:
```suggestion
export const LocaleDebug: React.FC = () => {
```

Fixes an eslint warning for missing explicit return type

Reply:
This shouldn't even be here, my mistake. Will remove.